public/request/game/update-meta.php

```diff
@@ -31,7 +31,8 @@
     return back()->withErrors(__('legacy.error.permissions'));
 }
 
-if (modifyGameData($user, $gameId, $input['developer'], $input['publisher'], $input['genre'], $input['release'], $input['guide_url'])) {
+if (modifyGameData($user, $gameId, $input['developer'], $input['publisher'], $input['genre'], $input['release'], $input['guide_url'])
+) {
```

Comment:
`) {` should be on the first line (or indented to indicate it's a continuation of a previous line)

Reply:
Oh boy, I feel bad wasting your time with formatting errors like this. Do we have a prettier config file or something like that to enforce these standards, that I can then set up my editor to auto-apply the rules for? 

Thanks for catching this, I've addressed it.